src/main/java/com/iexec/sms/tee/session/base/SecretSessionBaseService.java

```diff
@@ -354,12 +341,12 @@ List<Web2SecretHeader> getPostComputeSecretHeaders(TaskDescription taskDescripti
         return ids;
     }
 
-    public Map<String, String> getPostComputeEncryptionTokens(TeeSessionRequest request, String resultEncryptionKey)
+    Map<String, String> getPostComputeEncryptionTokens(final TeeSessionRequest request, final String resultEncryptionKey)
             throws TeeSessionGenerationException {
-        TaskDescription taskDescription = request.getTaskDescription();
-        String taskId = taskDescription.getChainTaskId();
-        Map<String, String> tokens = new HashMap<>();
-        boolean shouldEncrypt = taskDescription.isResultEncryption();
+        final TaskDescription taskDescription = request.getTaskDescription();
+        final String taskId = taskDescription.getChainTaskId();
+        final Map<String, String> tokens = new HashMap<>();
+        final boolean shouldEncrypt = taskDescription.isResultEncryption();
```

Comment:
The method isResultEncryption is deprecated.

Reply:
That's what I wrote about in Slack. This need to be addressed globally in a dedicated ticket.